salt/master.py

```diff
@@ -1950,6 +1950,7 @@ class ClearFuncs(TransportMethods):
         "mk_token",
         "wheel",
         "runner",
+        "_list_valid_minions",
```

Comment:
**question** does `_list_valid_minons` belong here in ClearFuncs? My *thought* is that it's fine here since it's not much different than `ping`, and is used from `publish`. But I would happily be educated as to why that thought is incorrect.